framework/test/mocha/fixtures/blocks.js

```diff
@@ -74,7 +74,26 @@ const GenesisBlock = stampit(Block, {
 	},
 });
 
+const BlockHeader = stampit({
```

Comment:
Do we need to use stampit necessarily here?

Reply:
Yes stampit is used for every fixture and should be used.